.projenrc.ts

```diff
@@ -21,8 +21,8 @@ const project = new typescript.TypeScriptProject({
     '@jsii/spec',
     'case',
     'fs-extra',
-    'glob-promise',
-    'glob',
+    'glob-promise@^5',
+    'glob@^8',
```

Comment:
Let's see if that passes, but if we are going down the route of keeping `glob-promise` please just update the versions in `package.json`

Not specifying a version constraint in `.projenrc.ts` is telling projen that the version is managed via `package.json`.

Reply:
Ah, apologies, I saw the note about package.json being autogenerated and thought I wasn't supposed to touch it. I'll change that, as it feels much less surprising.

Reply:
(it does pass, btw. I got the tests to run locally)